types/ramda/es/sequence.d.ts

```diff
@@ -0,0 +1,2 @@
+import { sequence } from '../index';
```

Comment:
this file and `src/sequence.d.ts` and other `sequence` related things should be removed too.